### PR TITLE
Sync UI and add change logs of UI

### PR DESCRIPTION
**Pull request by wilsonwu**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [ ] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -104,6 +104,9 @@ Release Notes.
 * Add envoy proxy component logo.
 * Chore: set up license-eye to check license headers and add missing license headers.
 * Fix prop for instances-survey and endpoints-survey.
+* Fix envoy icon in topology.
+* Fix the service logs implementation on UI.
+* Fix change the flask icon to light version for a better view of topology dark theme.
```
Comment (wu-sheng) [inline CHANGES.md:109]:
```suggestion
* Change the flask icon to light version for a better view of topology dark theme.
```

**Review by wu-sheng — COMMENTED** [review]
Your branch is not updated, please sync it or make us sync-able.

File: CHANGES.md (hunk 1 above)
Comment (wilsonwu) [reply CHANGES.md:109]: Allowed editable.

File: CHANGES.md [hunk 2]
```diff
@@ -104,6 +104,9 @@ Release Notes.
 * Add envoy proxy component logo.
 * Chore: set up license-eye to check license headers and add missing license headers.
 * Fix prop for instances-survey and endpoints-survey.
+* Fix envoy icon in topology.
+* Fix the service logs implementation on UI.
```
Comment (kezhenxu94) [inline CHANGES.md:108]:
```suggestion
* Implement the service logs on UI.
```

**Comment by wu-sheng** [comment]
FYI @Fine0830 Your previous log feature will be included in this sync.

**Comment by wu-sheng** [comment]
@wilsonwu We need you to fix this.

image

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**